modules/local/chai_1/main.nf

```diff
@@ -1,6 +1,5 @@
 process CHAI_1 {
     tag "$meta.id"
-    label 'process_high'
```

Comment:
Why this change @FloWuenne ? We will need higher resources to run on CPUs right? Maybe we need to customise the resource requests for CPU vs GPU instead?

Reply:
Fixed in https://github.com/seqeralabs/nf-chai/pull/14/commits/9c5bc4ae48468f94ace6c0d1c79472b77fba8d58